tests/main/interfaces-time-control/task.yaml

```diff
@@ -16,11 +16,14 @@ prepare: |
 
 restore: |
     # Restore the initial rtc configuration
+    snap connect test-snapd-timedate-control-consumer:netlink-audit
+    snap connect test-snapd-timedate-control-consumer:time-control
     if [ -f rtc.txt ]; then
-        timedatectl set-local-rtc "$(cat rtc.txt)"
+        test-snapd-timedate-control-consumer.timedatectl-time set-local-rtc "$(cat rtc.txt)"
     fi
 
-    date "$(cat now.txt)"
+    # TODO: re-enable once the ntp issue is fixed
+    #test-snapd-timedate-control-consumer.date "$(cat now.txt)"
```

Comment:
so can we use timedatectl set-time instead then?

Reply:
I tried that, but when the problem comes when the test does:
test-snapd-timedate-control-consumer.date "$nice_when"